tests/loading_from_clipboard/loading_from_clipboard_test.cpp

```diff
@@ -0,0 +1,66 @@
+#include "gtest/gtest.h"
+#include "loading_from_clipboard_test.h"
+#include <QTextStream>
+#include "../../common/decklist.h"
+
+DeckList *fromClipboard(QString *clipboard);
+
+DeckList *fromClipboard(QString *clipboard) {
+    DeckList *deckList = new DeckList;
+    QTextStream *stream = new QTextStream(clipboard);
+    deckList->loadFromStream_Plain(*stream);
+    return deckList;
+}
+
+struct AssertDeckList {
+    QList<QString> expectedCards;
+    QList<QString> actualCards;
+
+    AssertDeckList(QList<QString> _expectedCards) : expectedCards(_expectedCards) {}
```

Comment:
TIL C++ has implicit conversion. In this case I'd like to mark it as explicit because I don't know of a use case where implicit conversion would be useful for `AssertDeckList`.